packages/solidity-contracts/.gitignore

```diff
@@ -1,7 +1,7 @@
 node_modules
 .env
 .vscode
-.openzeppelin
+.openzeppelin/unknown*
```

Comment:
That's a strange entry

Reply:
It comes from OZ and unfortunately there is no way to customize it.

More info: https://docs.openzeppelin.com/upgrades-plugins/1.x/network-files

Quoting:

> It is advised that you commit to source control the files for all networks except the development ones (you may see them as .openzeppelin/unknown-*.json).